js/src/events/Main.js

```diff
@@ -15,6 +15,8 @@ import BookEvent from './Book'
 import DonateToEvent from './Donate'
 import WaitingList from './WaitingList'
 import Thanks from '../donations/Thanks'
+import YoutubePlayer from '../general/YoutubePlayer'
```

Comment:
I don't think you've included this file in git which is why CI is failing.

Best to read the errors if CI fails before asking for a review.

Reply:
Dunno what happened there been trying out a graphical git client.
Shall we add failure notifications to travis?